src/interface/src/app/shared/nav-bar/nav-bar.component.ts

```diff
@@ -24,11 +27,19 @@ export class NavBarComponent {
   hasSharePlanFeatureFLag =
```

Comment:
do you mean for this to have a capital L in Flag?

Reply:
thanks!